app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -178,23 +179,29 @@ class MainActivity : AppCompatActivity(),
             }
         }
 
-        aztec.setOverlay(predicate, 0, ColorDrawable(0x80000000.toInt()), Gravity.FILL, attrs)
-        val progressDrawable = resources.getDrawable(android.R.drawable.progress_horizontal)
+        aztec.setOverlay(predicate, 0, ColorDrawable(0x80000000.toInt()), Gravity.FILL)
+        aztec.updateElementAttributes(predicate, attrs)
```

Comment:
Thanks! I think that the performance hit from looping over every element is larger than the redundant setting of the same attributes but, this is not something I would block this PR over. Plus, I like the `Single Responsibility Principle` too so, 👍  😃